packages/rum/src/domain/record/record.ts

```diff
@@ -57,6 +58,24 @@ export function record(options: RecordOptions): RecordAPI {
     })
   }
 
+  if (isExperimentalFeatureEnabled('visualviewport')) {
+    if (visualViewport) {
+      const viewport = visualViewport
```

Comment:
**question:** Why renaming here?
**sugestion:** remove rename or in the first name the variable `visualViewport` as `viewport`

Reply:
TBH, I'm less certain here.

I believe our code minifier (Terser) won't minify global variables. Instead if we use an intermediate variable (line 63) I believe we can save 112 bytes (and move line 63 to line 60) without reducing readability- though it isn't conventional.

Reply:
Refactored to use this inside `viewports.ts`, saving perhaps 280 bytes.